backend/Tournament/serializers.py

```diff
@@ -19,12 +19,20 @@ class Meta:
 		fields = ['id', 'user', 'display_name']
 
 class TournamentMatchSerializer(serializers.ModelSerializer):
+	tournament_match_id = serializers.SerializerMethodField()
+	match_id = serializers.IntegerField(source='id', read_only=True)
+	state = serializers.CharField(source='get_state_display', read_only=True)
 	tournament_player_left = serializers.SerializerMethodField()
 	tournament_player_right = serializers.SerializerMethodField()
 	winner = serializers.SerializerMethodField()
 	class Meta:
 		model = Match
-		fields = ['id', 'tournament_player_left', 'tournament_player_right', 'winner']
+		fields = ['tournament_match_id', 'match_id', 'state', 'tournament_player_left', 'tournament_player_right', 'winner']
```

Comment:
for the individual match_id, id is enough, I would say. Although, can't hurt to be explicit, I suppose